dockers/docker-pde/base_image_files/port_breakout.py

```diff
@@ -0,0 +1,833 @@
+#! /usr/bin/python -u
+
+import os
+import sys
+import re
+import getopt
+import copy
+import json
+import shutil
+import os.path
+import subprocess
+
+SIM_HOST = False
+
+SAI_PROFILE_DELIMITER = '='
+INTERFACE_KEY="Ethernet"
+if not SIM_HOST:
+    NEW_FILE_EXT=""
+else:
+    NEW_FILE_EXT=""
+
+sonic_platforms = {
+    "x86_64-accton_as9716_32d-r0": {
+        "breakout": {
+            "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56,60,64,68,72,76,80,84,88,92,96,100,104,108,112,116,120,124": [ "1x400", "4x100", "4x50", "2x100", "2x200", "4x25", "4x10" ]
+        }
+    },
+    "x86_64-accton_as7326_56x-r0": {
+        "breakout": {
+            "48,52,56,60,64,68,72": [ "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-accton_as7712_32x-r0": {
+        "breakout": {
+            "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56,60,64,68,72,76,80,84,88,92,96,100,104,108,112,116,120,124": [ "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-accton_as7816_64x-r0": {
+        "breakout": {
+            "0,8,16,24,32,40,48,56,64,72,80,88,96,104,112,120": [ "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-accton_as7726_32x-r0": {
+        "breakout": {
+            "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56,60,64,68,72,76,80,84,88,92,96,100,104,108,112,116,120": [ "4x25", "4x10", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-delta_ag9032v1-r0": {
+        "breakout": {
+            "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56,60,64,68,72,76,80,84,88,92,96,100,104,108,112,116,120,124": [ "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-quanta_ix4_bwde-r0": {
+        "breakout": {
+            "128,132,136,140,144,148,152,156,192,196,200,204,208,212,216,220": [
+                "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-quanta_ix8_rglbmc-r0": {
+        "breakout": {
+            "48,52,56,60,64,68,72": [ "4x10", "4x25", "1x100", "1x40" ]
+        }
+    },
+    "x86_64-quanta_ix9_bwde-r0": {
+        "breakout": {
+             "0,8,16,24,32,40,48,56,64,72,80,88,96,104,112,120,128,136,144,152,160,168,176,184,192,200,208,216,224,232,240,248": [ "1x400", "4x100", "4x25", "4x10", "4x50", "2x200", "2x100" ]
+        }
+    }
+}
+
+#####################################################################################################
+### Platform related code
+
+if not SIM_HOST:
+    def get_platform():
+        cmd = "cat /host/machine.conf | grep onie_platform | cut -d '=' -f 2"
+        pin = subprocess.Popen(cmd,
```

Comment:
Variable name `pin` is used here, whereas `proc` is used in functions below. I prefer to make them all the same, and I prefer `proc`, as `pin` isn't as descriptive.